CHANGELOG.md

```diff
@@ -331,7 +331,7 @@
 
 ### Merged PRs:
 
-- [#207](https://github.com/TokTok/c-toxcore/pull/207) docs: correct instructions for cloning & harden agains repo name changes
```

Comment:
The changelog is generated, so this will get overwritten on the next release. Please make the change in the PR (https://github.com/TokTok/c-toxcore/pull/207) itself.